.travis.yml

```diff
@@ -0,0 +1,17 @@
+dist: trusty
+sudo: required
+language: c
+compiler:
+  - gcc
+  - clang
+
+script:
+  - make NANOPB_DIR=nanopb all
+  - make NANOPB_DIR=nanopb test
```

Comment:
it looks like "make test" doesn't do anything at the moment, presumably that will be fixed soon :)